### PR TITLE
DPB Breakout Cli Test with proper setup

### PR DESCRIPTION
**Pull request by samaity**
Signed-off-by: Sangita Maity <sangitamaity0211@gmail.com>

**- What I did**

1. Created "platform.json" for vs docker.
2. Upgrade click to 7.0.0 for vs docker.
3. Added Test case for Breakout Cli

This PR is dependent on [sonic-buildimage/pull/17](https://github.com/zhenggen-xu/sonic-buildimage/pull/17) and [sonic-swss/pull/17](https://github.com/zhenggen-xu/sonic-swss/pull/17)

**- How to verify it**
```


samaity@server09:~/REPO_FACTORY/GIT_REPO/DPB/sonic-buildimage/platform/vs/tests/breakout$ sudo pytest -s -v --dvsname=vs-sang test_breakout_cli.py
=================================================================== test session starts ====================================================================
platform linux2 -- Python 2.7.15+, pytest-3.3.0, py-1.8.0, pluggy-0.6.0 -- /usr/bin/python
cachedir: .cache
rootdir: /home/samaity/REPO_FACTORY/GIT_REPO/DPB/sonic-buildimage/platform/vs/tests/breakout, inifile:
collected 2 items

test_breakout_cli.py::TestBreakoutCli::test_InitialBreakoutMode remove extra link dummy
PASSED                                                                               [ 50%]
test_breakout_cli.py::TestBreakoutCli::test_breakout_modes **** Breakout Cli test Starts ****
**** 1X100G --> 2x50G passed ****
**** 2x50G --> 1x100G[40G] passed ****
**** 1X100G --> 4x25G[10G] passed ****
**** 4x25G[10G] --> 1x100G[40G] passed ****
**** 1X100G --> 2x50G mode change ****
**** 2X50G --> 4x25G[10G] passed ****
**** 4x25G[10G] --> 2X50G passed ****
**** 2x50G  -- > 1X100G mode change ****
**** 1x100G[40G] --> 2x25G(2)+1x50G(2) passed ****
**** 2x25G(2)+1x50G(2) --> 1x100G[40G] passed ****
**** 1x100G[40G] --> 1x50G(2)+2x25G(2)  passed ****
**** 1x50G(2)+2x25G(2) --> 1x100G[40G] passed ****
**** 1x100G[40G] --> 2x50G  mode change ****
**** 2x50G --> 2x25G(2)+1x50G(2)  passed ****
**** 1x50G(2)+2x25G(2) --> 2x25G(2)+1x50G(2)  passed ****
**** 2x25G(2)+1x50G(2)  --> 1x100G[40G]  passed ****
PASSED                                                                                    [100%]

```

### CODE REVIEW

File: platform/vs/tests/breakout/test_breakout_cli.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+            assert(status == True)
+
+            brkout_mode = fvs[0][1]
+            output_dict[key] = brkout_mode
+        output = collections.OrderedDict(sorted(output_dict.items(), key=lambda t: t[0]))
+        expected_dict = {'Ethernet8': '1x100G[40G]', 'Ethernet0': '1x100G[40G]', 'Ethernet4': '1x100G[40G]', 'Ethernet108': '1x100G[40G]', 'Ethernet100': '1x100G[40G]', 'Ethernet104': '1x100G[40G]', 'Ethernet68': '1x100G[40G]', 'Ethernet96': '1x100G[40G]', 'Ethernet124': '1x100G[40G]', 'Ethernet92': '1x100G[40G]', 'Ethernet120': '1x100G[40G]', 'Ethernet52': '1x100G[40G]', 'Ethernet56': '1x100G[40G]', 'Ethernet76': '1x100G[40G]', 'Ethernet72': '1x100G[40G]', 'Ethernet32': '1x100G[40G]', 'Ethernet16': '1x100G[40G]', 'Ethernet36': '1x100G[40G]', 'Ethernet12': '1x100G[40G]', 'Ethernet28': '1x100G[40G]', 'Ethernet88': '1x100G[40G]', 'Ethernet116': '1x100G[40G]', 'Ethernet80': '1x100G[40G]', 'Ethernet112': '1x100G[40G]', 'Ethernet84': '1x100G[40G]', 'Ethernet48': '1x100G[40G]', 'Ethernet44': '1x100G[40G]', 'Ethernet40': '1x100G[40G]', 'Ethernet64': '1x100G[40G]', 'Ethernet60': '1x100G[40G]', 'Ethernet20': '1x100G[40G]', 'Ethernet24': '1x100G[40G]'}
```
Comment (zhenggen-xu) [inline platform/vs/tests/breakout/test_breakout_cli.py]: rewind the line

File: platform/vs/tests/breakout/test_breakout_cli.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        expected = collections.OrderedDict(sorted(expected_dict.items(), key=lambda t: t[0]))
+        assert output == expected
+
+    # Breakout Cli Test Mode
+    def test_mode_1X100G(self, dvs):
```
Comment (zhenggen-xu) [inline platform/vs/tests/breakout/test_breakout_cli.py]: change the name to test_breakout_modes()

File: platform/vs/tests/breakout/test_breakout_cli.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        expected = self.read_Json(dvs)
```
Comment (zhenggen-xu) [inline platform/vs/tests/breakout/test_breakout_cli.py:72]: Add 2x50G -> 4x25 and 4x25G -> 2x50G

File: platform/vs/tests/breakout/test_breakout_cli.py (hunk 1 above)
Comment (samaity) [reply platform/vs/tests/breakout/test_breakout_cli.py]: addressed

File: platform/vs/tests/breakout/test_breakout_cli.py (hunk 2 above)
Comment (samaity) [reply platform/vs/tests/breakout/test_breakout_cli.py]: addressed

File: platform/vs/tests/breakout/test_breakout_cli.py (hunk 3 above)
Comment (samaity) [reply platform/vs/tests/breakout/test_breakout_cli.py:72]: added these test cases.

**Review by zhenggen-xu — APPROVED** [review]

**Pull request merged**